cost/azure/hybrid_use_benefit/README.md

```diff
@@ -1,49 +1,35 @@
-## Azure Hybrid Use Benefit Policy
+# Azure Hybrid Use Benefit Policy
 
-### What it does
+## What it does
 
 This Policy Template is used to automatically apply the Azure Hybrid Use Benefit (AHUB) to all eligible VMs in an Azure Subscription.
 
-### Prerequisites
+## Functional Details
 
-- Azure Service Principal (AKA Azure Active Directory Application) with the appropriate permissions to manage resources in the target subscription
-- The following RightScale Credentials
-  - `AZURE_APPLICATION_ID`
-  - `AZURE_APPLICATION_KEY`
-- The `policy_designer`, `policy_manager`, `credential_viewer` & `policy_publisher` roles
-
-### Azure Required Permissions
-
-- Microsoft.Compute/virtualMachines/read
-- Microsoft.Compute/virtualMachines/write
-
-### Installation
-
-1. Follow steps to [Create an Azure Active Directory Application] (https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
-1. Grant the Azure AD Application access to the necessary subscription(s)
-1. [Retrieve the Application ID & Authentication Key] (https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
-1. [Retrieve your Tenant ID] (https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
-
-### Functional Details
-
-_ The policy identifies all Windows server instances that are not currently using [Azure Hybrid Use Benefit] (https://azure.microsoft.com/en-us/pricing/hybrid-benefit/). It raises an incident for all applicable VMs not currently using AHUB, which once approved, will enable AHUB on all identified instances.
```

Comment:
The text doesn't have a link. check the markdown.

Reply:
Addressed the issue.